cdshealpix/ring/healpix.py

```diff
@@ -187,7 +202,7 @@ def healpix_to_skycoord(ipix, nside, dx=0.5, dy=0.5):
     ----------
     ipix : `numpy.array`
         The HEALPix cell indexes given as a `np.uint64` numpy array.
-    nside : int
+    nside : `numpy.array`
```

Comment:
Write `numpy.ndarray` as type in docstrings everywhere. That's the type. `numpy.array` is a function that casts lists etc to `ndarray`.